pkgs/top-level/all-packages.nix

```diff
@@ -9471,6 +9471,8 @@ with pkgs;
     libpng = libpng12;
   };
 
+  pngpaste = callPackage ../tools/graphics/pngpaste { };
```

Comment:
```suggestion
  pngpaste = callPackage ../tools/graphics/pngpaste {
    inherit (darwin.apple_sdk.frameworks) AppKit Cocoa;
  };
```

Reply:
Done. Out of interest, why is this the preferred pattern?